### DISCUSSION TITLE
Performance - 3

### DISCUSSION THREAD

**Discussion by changjurhee** [changjurhee/AONE#15] [Polls]
## Quality Attributes

Type | Description | QA Scenario (Stimulus / Source of stiulus / Artifact / Environment / Response / Response measure) | Tatics
 -- | -- | -- | --
Performance | Adjust bandwidth use as necessary to maintain communication quality. (latency) | Even if the network status of the gateway deteriorates, video communication is turned off so that voice communication operates without delay of more than 1 second.<br>- Deteriorating network conditions<br>- Gateway<br>- Call service<br>- Runtime<br>- Turn off video communication so that voice communication is maintained<br>- Latency is less than 1 second |

**Comment by ju297** [comment]
Manage work requests - 대역폭이 충분하지 못할 경우 영상 처리에 대한 처리를 미루거나 비활성화 시키는 방법

**Comment by codekistone** [comment]
reduce overhead - If the bandwidth is insufficient, reduce the transmission quality of the transmission video or make it operate as voice only so that there is no disconnection of the call connection.

**Comment by changjurhee** [comment]
- 성능 항목에 맞게끔 설명을 보완 하자,.
